cloud_functions/flagCryptominer.js

```diff
@@ -4,11 +4,20 @@ const utils = require('./utils')
 
 const run = utils.withErrorReporting((mixpanel, message) => {
   const userSubjectId = validate(message)
-  return promisify(mixpanel.people.set)(`google:${userSubjectId}`, { 'isCryptominer': true })
+  return new Promise((resolve, reject) => {
+    mixpanel.people.set(`google:${userSubjectId}`, { 'isCryptominer': true },
+        error => {
+          if (error) {
+            reject(error)
+          } else {
+            resolve()
+          }
+        })
+  })
 })
 
 const validate = message => {
-  if (!(message.attributes && message.attributes.cryptominer === true)) {
+  if (!(message.attributes && (message.attributes.cryptominer === true || message.attributes.cryptominer === 'true'))) {
```

Comment:
This feels sloppy. Can we just use one correct value?

Reply:
Yeah, I agree that it's sloppy. I can't find anything definitive about the types of these attributes. I was testing this by posting messages to the topic in cloud console, and that didn't give me any control over the value type; everything was coming through as a string.